### PR TITLE
Upgrade Decidim to release/0.21-stable

### PR DESCRIPTION
**Pull request by victorol1**
**MERGE #47 FIRST**
Upgrade Decidim to release/0.21-stable. Apply upgrade notes after deploying: https://github.com/decidim/decidim/blob/0.21-stable/CHANGELOG.md#upgrade-notes-1

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 ruby RUBY_VERSION
-DECIDIM_VERSION = { git: 'https://github.com/decidim/decidim.git', branch: '0.20-stable' }
+DECIDIM_VERSION = { git: 'https://github.com/decidim/decidim.git', branch: '0.21-stable' }
```
Comment (tramuntanal) [inline Gemfile:4]:
```suggestion
DECIDIM_VERSION = { git: 'https://github.com/decidim/decidim.git', branch: 'release/0.21-stable' }
```

**Comment by tramuntanal** [comment]
@victorol1 some conflicts appeared, can you check please?

**Comment by victorol1** [comment]
> @victorol1 some conflicts appeared, can you check please?

Done! Conflicts solved.

**Comment by tramuntanal** [comment]
We must remember to set the timezone to Madrid for all organizations

**Pull request merged**